geopolars/Cargo.toml

```diff
@@ -40,7 +40,6 @@ thiserror = "1.0"
 
 proj = { version = "0.27.0", optional = true, features = [
   "geo-types",
-  "bundled_proj",
```

Comment:
Is this just for testing or do you prefer not using this flag?

Reply:
This was the only way I found to be able to find the proj_data directory - by using the systems proj installation. This also means we need to compile proj ourselves in the CI.

The outcome in the end changes little, though.[proj will still be statically linked to geopolars in the wheels](https://github.com/georust/proj/blob/d7185a7152db99ff34775e76064bc52c3ef45bd5/proj-sys/build.rs#L118).

Reply:
I'm ok with compiling PROJ ourselves in CI. We might still want to add a new top-level feature so that Rust consumers of `geopolars` would be able to opt in to bundled proj? I.e. if they don't have proj installed locally

Reply:
Good point. Added in d271a69b06a9d57e40042d8d9299618045aa97f5